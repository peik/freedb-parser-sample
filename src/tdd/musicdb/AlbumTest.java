package tdd.musicdb;

import org.approvaltests.Approvals;
import org.junit.Test;

public class AlbumTest {

	@Test
	public void testListing() throws Exception {
		Album a = new Album();
		a.setTitle("Adele / 21");
		a.setYear(2011);
		a.setGenre("blues");
		a.setDiscId("a1234567");
		a.addSong(new Song("Rolling in the Deep", 229));
		a.addSong(new Song("Rumour Has It", 223));
		Approvals.approve(a.getListing());
	}
}
