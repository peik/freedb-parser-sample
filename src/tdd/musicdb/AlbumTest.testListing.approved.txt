Adele / 21	Details
Tracks: 2
Total time: 7:32
Year: 2011
Disc-ID: blues / a1234567

1. Rolling in the Deep	3:49
2. Rumour Has It	3:43
